tests/integration_tests/test_empty_value_handling.py

```
@@ -46,19 +47,17 @@ def test_empty_value_handling(self, db_client):
             # Test 5: Mixed empty and non-empty values in batch operations
             self._test_mixed_empty_values_batch(collection)
 
-            print(f"✅ All empty value tests passed")
+            print("✅ All empty value tests passed")
         finally:
             # Cleanup
             try:
                 db_client.delete_collection(name=collection_name)
             except Exception as cleanup_error:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)